primitives/src/vid.rs

```diff
@@ -84,6 +83,14 @@ pub struct VidDisperse<V: VidScheme + ?Sized> {
     pub commit: V::Commit,
 }
 
+/// Trait for [`VidScheme::Common`].
+pub trait CommonBounds:
+    Clone + Debug + DeserializeOwned + Eq + PartialEq + Hash + Serialize + Sync
+{
+    /// Get the payload byte length.
+    fn payload_byte_len(&self) -> usize;
```

Comment:
Any reason for creating a separate trait rather than putting `payload_byte_len()` in the VidScheme trait?

Reply:
Agree that it is a natural API for `Common`, but it's also OK to think it as an API of the VID instantiation. As each VID instantiation/execution is also parameterized by the payload/block size to be dispersed. That said, it might be simpler to put it directly in `VidScheme.`